master-firmware/src/strategy/state.cpp

```diff
@@ -6,10 +6,13 @@ RobotState initial_state(void)
 {
     RobotState state;
     state = RobotState_init_default;
-    for (size_t i = 0; i < 3; i++)
+
+    size_t num_pucks = sizeof(state.puck_available) / sizeof(bool);
```

Comment:
Does this work in a packed context ? Not sure.

Reply:
Looking at the generated code, it's a regular C array of booleans
```
bool puck_available[12];
```
so it's fine. Also the `packed=true` doesn't change the generated code, so I guess it's redundant.